vpdq/python/vpdq.pyx

```diff
@@ -78,14 +76,14 @@ def hash_to_hex(hash_value: "Hash256"):
     Returns:
         str: hex str of hash
     """
-    return hashToString(hash_value)
+    return str(hashToString(hash_value), "utf-8")
 
 
 def str_to_hash(str_hash: str):
     return fromStringOrDie(str(str_hash).encode("utf-8"))
```

Comment:
According to your typing, you are passing in `str`, so why cast to str?

Assume the return type is Hash256

Reply:
The return type is binary literal string. For example, a hash str "000ffff...." will be "b'000ffff.....".